packages/web-components/fast-components/src/slider/fixtures/base.html

```diff
@@ -46,7 +46,11 @@ <h4>horizontal with left offset</h4>
 
         <!-- Margin offsets vertical -->
         <h4>Vertical with top offset</h4>
-        <fast-slider orientation="vertical" , style="margin-left: 100px;"></fast-slider>
+        <fast-slider
+            orientation="vertical"
+            ,
```

Comment:
Noticed this while running this just now, I don't think this comma is meant to be an attribute :).